pkg/crc/machine/machine.go

```diff
@@ -305,9 +305,13 @@ func Start(startConfig StartConfig) (StartResult, error) {
 	// If no error, return usage message
 	if result.Error == "" {
 		time.Sleep(time.Minute * 3)
-		logging.Infof("To access the cluster using 'oc', run 'eval $(crc oc-env) && oc login -u kubeadmin -p %s %s'", result.ClusterConfig.KubeAdminPass, result.ClusterConfig.ClusterAPI)
-		logging.Infof("Access the OpenShift web-console here: %s", result.ClusterConfig.WebConsoleURL)
-		logging.Infof("Login to the console with user: kubeadmin, password: %s", result.ClusterConfig.KubeAdminPass)
+		logging.Infof("")
+		logging.Infof("To access the cluster, first set up your environment by following 'crc oc-env' instructions")
+		logging.Infof("Then you can access it by running 'oc login -u developer -p developer %s'", result.ClusterConfig.ClusterAPI)
+		logging.Infof("To login as an admin, username is 'kubeadmin' and password is %s", result.ClusterConfig.KubeAdminPass)
+		logging.Infof("")
+		logging.Infof("Access the OpenShift web console here: %s", result.ClusterConfig.WebConsoleURL)
+		logging.Infof("Login to the console with user 'kubeadmin' and the password above")
```

Comment:
maybe we can remove the line `Login to the console with user 'kubeadmin' and the password above` both user accounts can be used to login to conosle

Reply:
Ah, I did not know that. Maybe change the last 2 lines to
"These credentials can also be used to access the OpenShift web console at %s"?